### PR TITLE
Pokemon modal structure and basic styles

### LINKED ISSUE #18: Create Modal component to display a Pokemon's info.
**Is your feature request related to a problem? Please describe.**
Make modal to display Pokemon name, type, sprite, dex number, background, and Tabs component.

**Describe the solution you'd like**
Use hooks to open and close modal, fill modal with fetched data.

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Basic layout and structure for modal
2. Basic modal functionality

## Purpose
Modal opens and closes displaying pokemon info to user

Closes #18

### CODE REVIEW

**Review by gonzalespaulb — APPROVED** [review]
LGTM

File: src/components/PokeCard.js [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+        </li>
       </ul>
+      <Modal
+        className="modalWindow"
+        isOpen={modalIsOpen}
+        onRequestClose={() => {
+          return setModalIsOpen(false);
+        }}>
+        <CustomModal
+          setIsOpen={setModalIsOpen}
```
Comment (kmausolf) [inline src/components/PokeCard.js:30]: Does this work without passing a parameter to the setter?

Comment (dannysanchez559) [reply src/components/PokeCard.js:30]: No it doesn't...  This is a custom React-Modal package we installed.

File: src/components/Modal.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          </div>
+
+          <div className="modalTabContent">
+            <p>**Content goes in here **</p>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
```
Comment (BlackCubes) [inline src/components/Modal.js:50]: I'm curious how this is going to look 🍺

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-modal": "^3.14.3",
```
Comment (BlackCubes) [inline package.json:11]: Good package!

**Review by JorgeLVilla — APPROVED** [review]
Excited to see the modal live!

File: src/components/PokeCard.js (hunk 1 above)
Comment (dannysanchez559) [reply src/components/PokeCard.js:30]: And the React-Modal must have the <Modal> tag while our Modal was renamed to <CustomModal>

**Review by BlackCubes — APPROVED** [review]
Looks good, and I am curious how everything is going to look

File: src/util/fetchPokemonData.js [hunk 4]
```diff
@@ -13,6 +14,17 @@ export const fetchPokemon = async (nameOrId) => {
   }
 };
 
+// make function for getting flavor text and pokemon color using speciesUrl
+export const fetchSpecies = async (nameOrId) => {
```
Comment (krferrell) [inline src/util/fetchPokemonData.js:18]: Are you all using the Js wrapper? There are built-in functions and caching built-in.

**Review by krferrell — APPROVED** [review]

File: src/util/fetchPokemonData.js (hunk 4 above)
Comment (dannysanchez559) [reply src/util/fetchPokemonData.js:18]: Yes Kenny, we are using JS wrapper, that script is in our index.html, and the file is in the root of our project.

**Pull request merged**